openedx/adg/common/mailchimp_pipeline/tests/test_handlers.py

```diff
@@ -152,12 +153,15 @@ def test_send_user_enrollments_to_mailchimp_with_valid_enrollment_event(mocker,
     """
     event = EnrollStatusChange.enroll
     user = UserFactory()
-    dummy_kwargs = {'user': user, 'course_id': 'test_course'}
-    mock_call = mocker.patch.object(mailchimp_handlers, 'task_send_user_enrollments_to_mailchimp')
+    course = CourseOverviewFactory()
 
-    mailchimp_handlers.send_user_enrollments_to_mailchimp(sender=None, event=event, **dummy_kwargs)
+    dummy_kwargs = {'user': user, 'course_id': course.id}
+    mock_get_or_create = mocker.patch.object(mailchimp_handlers.CourseMeta.objects, 'get_or_create')
+    mock_enrolls = mocker.patch.object(mailchimp_handlers, 'get_enrollment_course_names_and_short_ids_by_user')
+    mock_enrolls.return_value = '100, 101, 102', 'course1, course2, course3'
```

Comment:
Lets update `'100, 101, 102', 'course1, course2, course3'`

to `'100,101,102', 'course1,course2,course3'`